DataAPI/Startup.cs

```diff
@@ -15,9 +18,16 @@ public Startup(IConfiguration configuration)
 
         public IConfiguration Configuration { get; }
 
-        // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            services.AddAutoMapper(typeof(Startup));
+            services.AddScoped<DealsRepository>(); //TODO: rename to singular
```

Comment:
Should we mark this as a "TODO" issue in Github?

Reply:
That was just a silly comment that I forgot